# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: accessible/base/CacheConstants.h
# Commit: 09de143215a1
# Full Hash: 09de143215a13a22dcb2cd53d4ed1dfb60480c82
# Author: Atila Butkovits <abutkovits@mozilla.com>
# Date: 2025-09-30 02:48 +0300
# Regressor Bug: 1987945
# File Overlap Count: 2
# Overlapping Files: accessible/base/nsCoreUtils.cpp, accessible/tests/browser/relations/browser_anchor_positioning.js
# Description:
#   Revert "Bug 1987945: apply code formatting via Lando" for causing bustages at DocAccessible.cpp.
# ==============================================================================

diff -r 826eb4ac00dd -r 09de143215a1 accessible/base/CacheConstants.h
--- a/accessible/base/CacheConstants.h	Tue Sep 30 02:35:22 2025 +0300
+++ b/accessible/base/CacheConstants.h	Tue Sep 30 02:48:50 2025 +0300
@@ -87,12 +87,6 @@
      RelationType::FLOWS_FROM},
     {nsGkAtoms::aria_details, nullptr, RelationType::DETAILS,
      RelationType::DETAILS_FOR},
-    {nsGkAtoms::commandfor, nullptr, RelationType::DETAILS,
-     RelationType::DETAILS_FOR},
-    {nsGkAtoms::popovertarget, nullptr, RelationType::DETAILS,
-     RelationType::DETAILS_FOR},
-    {nsGkAtoms::target, nullptr, RelationType::DETAILS,
-     RelationType::DETAILS_FOR},
     {nsGkAtoms::aria_errormessage, nullptr, RelationType::ERRORMSG,
      RelationType::ERRORMSG_FOR},
 };